index.html

```diff
@@ -173,126 +173,445 @@ <h2>
+            regardless of lock state.
+          </p>
+        </p>
+      </section>
+      <section>
+        <h3>
+          `PointerLockOptions` dictionary
+        </h3>
+        <pre class="idl">
+          dictionary PointerLockOptions {
+            boolean unadjustedMovement = false;
+          };
+        </pre>
+        <dl data-dfn-for="PointerLockOptions" data-link-for=
+        "PointerLockOptions">
+          <dt>
+            <dfn>PointerLockOptions</dfn> dictionary
+          </dt>
+          <dd>
+            <p>
+              The options dictionary to customize how the pointer behaves in
+              the locked mode.
+            </p>
+          </dd>
+          <dt>
+            <dfn>unadjustedMovement</dfn> member
+          </dt>
+          <dd>
+            <p>
+              If this value is set to `true`, then the pointer movements will
+              not be affected by the underlying platform modications such as
+              mouse accelaration.
+            </p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h3>
+          `pointerlockchange` and `pointerlockerror` Events
+        </h3>
+        <p>
+          Two events are used to communicate pointer lock state change or an
+          error in changing state. They are named <dfn>pointerlockchange</dfn>
+          and <dfn>pointerlockerror</dfn>.
+        </p>
+        <p>
+          Magnification software increases the size of content on the screen.
+          It uses the mouse to move the magnified point of focus around. When a
+          pointer lock is initiated, the magnification software needs to switch
+          to using the keyboard to move the magnified point of focus around
+          instead. When a pointerlockchange event is fired, web browsers
+          therefore need to make sure the event is communicated to assistive
+          technologies like screen magnifiers.
+        </p>
+      </section>
+      <section>
+        <h3><dfn>Exit Pointer Lock</dfn></h3>
+        <p>
+          The process of exiting pointer lock, given an element, is as follows:
+          <ol>
+            <li>
+              The system mouse cursor must be displayed again and positioned at <a>cursor position</a>
+            </li>
+            <li>
+              <a href="https://html.spec.whatwg.org/multipage/webappapis.html#queue-an-element-task">
+                Queue an element task</a>, given this element, on the <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                  user Interaction task source</a> to do the following:
+                  <ol><li>
+                    <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                    <a>pointerlockchange</a> at the given element's
+                      <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>
+                  </li></ol>
```

Comment:
Please avoid hard links to the HTML spec. Please use the following and work with @domenic to export missing terms from HTML as needed:  

```suggestion
              [=Queue an element task=], given this element, on the [=user Interaction task source=] to do the following:
                  <ol><li>
                   [=Fire an event=] named `"pointerlockchange"` at the given element's [=node document=].
                  </li></ol>
```

Reply:
Hard links are removed hence marked as resolved.